### PR TITLE
Changing test assembly input type.

### PR DESCRIPTION
**Pull request by kaadhina**
Fix for Bug 329867:Its confusing for VSTest task to accept source filter as file path, when the actual value supported is only a regex based string

### CODE REVIEW

File: Tasks/VsTest/task.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   "inputs": [
     {
       "name": "testAssembly",
-      "type": "filePath",
+      "type": "string",
```
Comment (bryanmacfarlane) [inline Tasks/VsTest/task.json]: Should it be a multi line control so you don't need the semi colons?  Of course the task would still need some compat code and support splitting on existing lines.  See the publish artifacts task which takes a list of patterns

Comment (acesiddhu) [inline Tasks/VsTest/task.json]: any impact on compat

Comment (acesiddhu) [inline Tasks/VsTest/task.json]: looks fine

Comment (kaadhina) [inline Tasks/VsTest/task.json]: Going ahead with this fix. Leaving this as string for now.

**Pull request merged**